src/Service/PaymentSplitService.php

```diff
@@ -61,7 +70,7 @@ public function getTransfersFromOrders(array $orders): array
                 }
 
                 // Use existing transfer
-                $transfer = $this->stripeTransferFactory->updateFromOrder($transfer, $order);
+                $transfer = $this->stripeTransferFactory->updateFromOrder($transfer, $order, $pendingDebits);
```

Comment:
Instead of passing the full array, you could pass the relevant object mapped against the order ID.

Reply:
Done